datafusion/sqllogictest/test_files/arrow_typeof.slt

```diff
@@ -405,7 +405,7 @@ select arrow_cast([1], 'FixedSizeList(1, Int64)');
 ----
 [1]
 
-query error DataFusion error: Optimizer rule 'simplify_expressions' failed
+query error
```

Comment:
would it be possible to update the slt files with the new error message ? It would be easier to validate the test still makes sense

Reply:
Done.